xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/layouts/livedata-table.html

```diff
@@ -0,0 +1,90 @@
+<template>
+    <table class="xwiki-livetable">
```

Comment:
We use 2 space indendation for HTML, JavaScript and CSS in XWiki. It would be nice to keep  @the code style consistent.